docs/index-custom-title-page.html

```diff
@@ -0,0 +1,249 @@
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 16px;
+    text-align: left;
+  }
+
+  .card-description {
+    color: #000;
+  }
+
+  #guide a.no-text-decoration:hover {
+    text-decoration: none!important;
+  }
+
+  .icon {
+    width: 30px;
+    height: 30px;
+    background-position: bottom;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .section-title {
+    font-size: 22px;
+    margin-left: .5rem;
+    font-weight: 700;
+  }
+
+  .ul-col-1 {
+    columns: 1;
+    -webkit-columns: 1;
+    -moz-columns: 1;
+  }
+
+  @media (min-width:769px) {
+    .ul-col-md-2 {
+      columns: 2;
+      -webkit-columns: 2;
+      -moz-columns: 2;
+    }
+  }
+
+  @media screen and (max-width: 600px) {
+
+  }
+</style>
+
+<div class="legalnotice"></div>
+
+<div class="row my-4">
+  <div class="col-md-6 col-12">
+    <h2>Bring your data to life</h2>
+    <p>
+      Take this tutorial for the basics of exploring and visualizing your data.
+    </p>
+    <p>
+      <a href="get-started.html">
+        <button class="btn btn-primary">Create a visualization</button>
+      </a>
+    </p>
+    <p>
+      <a class="inline-block mr-3" href="https://www.elastic.co/webinars/getting-started-kibana">Demo</a>
+      <a class="inline-block mr-3" href="https://www.elastic.co/videos/training-how-to-series-stack">How-to videos</a>
+      <a class="inline-block mr-3" href="https://www.elastic.co/guide/en/kibana/8.4/whats-new.html">What's new</a>
+  </p>
+  </div>
+  <div class="col-md-6 col-12">
+    <img class="w-100" src="https://i.imgur.com/ew7ierC.png" />
+  </div>
+</div>
+
+<h3>Explore by use case</h3>
+
+<div class="row my-4">
+  <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/enterprise-search/current/start.html">
+      <div class="card">
+        <div class="icon" style="background-image: url('https://i.imgur.com/JUxHITA.png');"></div>
+        <h4>Search</h4>
+        <p class="card-description">Learn how to create a search experience for your content.</p>
+      </div>
+    </a>
+  </div>
+  <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html">
+      <div class="card">
+        <div class="icon" style="background-image: url('https://i.imgur.com/XCIZjs1.png');"></div>
+        <h4>Observe</h4>
+        <p class="card-description">Follow our guides for monitoring logs, metrics, and traces.</p>
+      </div>
+    </a>
+  </div>
+  <div class="col-md-4 col-12 mb-2">
+    <a class="no-text-decoration" href="https://www.elastic.co/guide/en/security/current/es-overview.html">
+      <div class="card">
+        <div class="icon" align-items-center style="background-image: url('https://i.imgur.com/MNgoZfx.png');"></div>
+        <h4>Protect</h4>
+        <p class="card-description">Everything you need to know about protecting your data.</p>
```

Comment:
"Everything you need to know about" seems a bit wordy, and we could use the space for more focused terminology. This revision borrows some phrasing from the app description on the UI's Home page:
```suggestion
        <p class="card-description">Learn how to protect your data and respond to threats across your infrastructure.</p>
```

<img width="800" alt="image" src="https://user-images.githubusercontent.com/87339667/189157641-d0c2f1ed-a331-421d-ab5b-c920ef0965a6.png">